tests/js/server/permissions/test-oasis.js

```diff
@@ -0,0 +1,70 @@
+/*jshint globalstrict:false, strict:false */
+/* global getOptions, assertTrue, assertFalse, assertEqual, assertMatch, fail, arango */
```

Comment:
it seems this line is copy&paste. Most of these symbols are not used in this file.

Reply:
Removed `arango` the others are global functions (`assertFalse`, `assertMatch`, `fail`) will probably be used sooner or later in this file.